### PR TITLE
Awx 13

### PR DESCRIPTION
**Pull request by cfelder**
Fix missing awx_ prefixes

* memcached socket
* kubernetes redis

Added new variables to overwrite awxtask and awxweb in one go:

* awx_awx_version: 13.0.0
* awx_awx_image: docker.io/ansible/awx:{{ awx_awx_version }}

Kept backward compatible variables:

* awx_awxweb_{image, version}
* awx_awxtask_{image, version}

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -4,12 +4,13 @@
...
-awx_awxweb_version: 10.0.0
-awx_awxtask_version: 10.0.0
+awx_awx_version: 13.0.0
+awx_awxweb_version: "{{ awx_awx_version }}"
+awx_awxtask_version: "{{ awx_awx_version }}"
 awx_postgres_version: 10
 
-awx_awxweb_image: docker.io/ansible/awx_web:{{ awx_awxweb_version }}
-awx_awxtask_image: docker.io/ansible/awx_task:{{ awx_awxtask_version }}
+awx_awxweb_image: "{{ awx_awx_image | default('docker.io/ansible/awx:{}'.format(awx_awxweb_version)) }}"
```
Comment (ikke-t) [inline defaults/main.yml:12]: Please help me understand what's the purpose to use so complicated notation with .default in a file that is supposed to have just defaults? what's the difference if you'd just have {{ awx_awx_image }}: {{ awx_awxweb_version }} ?

This syntax was new to me, glad to learn it. I just don't get what does it add here?

**Review by ikke-t — COMMENTED** [review]
Thanks, good stuff. I have a question however. Please help me understand what's the purpose in defaults.yaml to use .defaults?

File: defaults/main.yml (hunk 1 above)
Comment (cfelder) [reply defaults/main.yml:12]: When providing just a default for awx_awx_image you are loosing the previous notation. I wanted to keep backward compatibility.

If awx_awx_image is not defined one can use:
* ``awx_awxweb_{image, version}``
* ``awx_awxtask_{image, version}``

in playbooks as before

or

one can define just ``awx_awx_image`` to set both images, e.g.

``awx_awx_image: quay.io/cfelder/awx:nopreload-{{ awx_awx_version }}``.

Comment (ikke-t) [reply defaults/main.yml:12]: ok, thanks

**Pull request merged**